### PR TITLE
Create ajax_shoutbox.php

### PR DESCRIPTION
**Pull request by Mauron**
Italian translation for ajax_shoutbox.php

### CODE REVIEW

**Comment by paul999** [comment]
Translation is incomplete. Please provide one PR with the complete translation.

**Comment by Mauron** [comment]
Hi,
I translated the three files singularly, since I'm still not able to make a pull request of a whole folder.
Or could it be that I missed some line instead?

**Comment by paul999** [comment]
You need to create your own fork, and edit the files in a new branch. After that you can create a new PR with all the changes in one PR.

The reason for that is that I don't want to have incomplete translation merged. Having them in seperate PRs costs me a lot of extra time to make sure everything is complete.

**Comment by paul999** [comment]
I merged all your PRs into #26 There will be some more language changes coming weekend, are you able to make some changes before sunday?

**Comment by Mauron** [comment]
I think I can do that.
Sorry for the inconvenience about PRs, I'll try to make a single request for the next language update.

File: language/it/ajax_shoutbox.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+$lang = array_merge(
+	$lang, array(
+		'AJAX_SHOUTBOX'                  => 'Bacheca',
+		'AJAX_SHOUTBOX_MESSAGE'          => 'Scrivi un messaggio',
+		'AJAX_SHOUTBOX_ONLY_AJAX'        => 'Per poter utilizzare la bacheca è necessario abilitare JavaScript',
+		'AJAX_SHOUTBOX_NO_PERMISSION'    => 'Permessi insufficienti per l\'azione selezionata',
+		'AJAX_SHOUTBOX_MESSAGE_EMPTY'    => 'Messaggio vuoto',
+		'AJAX_SHOUTBOX_ERROR'            => 'Errore',
+		'AJAX_SHOUTBOX_MISSING_ID'       => 'Impossibile rimuovere il messaggio',
+	)
```
Comment (Mauron) [inline language/it/ajax_shoutbox.php]: 'AJAX_SHOUTBOX_NO_SUCH_POST'     => 'Impossibile trovare il messaggio',

File: language/it/ajax_shoutbox.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+);
```
Comment (Mauron) [inline language/it/ajax_shoutbox.php]: )

**Pull request closed without merging**